src/test/java/com/yahoo/bullet/querying/QuerierTest.java

```diff
@@ -902,15 +901,16 @@ public void testComputation() {
         Expression expression = ExpressionUtils.makeBinaryExpression(Expression.Operation.ADD,
                                                                      ExpressionUtils.makeLeafExpression(new Value(Value.Kind.FIELD, "a", Type.INTEGER)),
                                                                      ExpressionUtils.makeLeafExpression(new Value(Value.Kind.VALUE, "2", Type.LONG)));
-        String query = makePostAggregationsQuery(makeComputation(expression, "newName"));
+        String query = makeRawFullQuery("a", Arrays.asList("null"), Clause.Operation.NOT_EQUALS, Aggregation.Type.RAW, 500, Arrays.asList(makeComputation(expression, "newName")), Pair.of("b", "b"));
         Querier querier = make(Querier.Mode.ALL, query);
         querier.initialize();
 
-        IntStream.range(0, 4).forEach(i -> querier.consume(RecordBox.get().add("a", i).getRecord()));
+        IntStream.range(0, 4).forEach(i -> querier.consume(RecordBox.get().add("a", i).add("b", i).getRecord()));
 
         List<BulletRecord> result = querier.getResult().getRecords();
         Assert.assertEquals(result.size(), 4);
         Assert.assertEquals(result.get(0).get("newName"), 2L);
+        Assert.assertFalse(result.get(0).hasField("a"));
```

Comment:
Let's add some asserts for ```b``` in the results

Reply:
Done.